classes/class-wc-connect-shipping-label.php

```diff
@@ -202,9 +175,18 @@ public function get_selected_packages( WC_Order $order ) {
 							$formatted = WC_Connect_Compatibility::instance()->get_formatted_variation( $product, true );
 							$product_data['attributes'] = $formatted;
 						}
+						$customs_info = get_post_meta( $product_data['product_id'], 'customs_info', true );
```

Comment:
We should probably use a more unique meta_key, perhaps `wc_connect_customs_info`?

Reply:
Agreed, changed in https://github.com/Automattic/woocommerce-services/pull/1382/commits/f9e4f28cbf5117c8e08a2206b335020603063cf2